### PR TITLE
fix: ensure return types matches method definesions

### PR DESCRIPTION
**Pull request by boajer**
https://oat-sa.atlassian.net/browse/ADF-661

### CODE REVIEW

File: models/classes/taskQueue/TaskLog/Broker/RdsTaskLogBroker.php [hunk 1]
```diff
@@ -212,15 +212,15 @@ public function getStats(TaskLogFilter $filter): TasksLogsStats
      */
     public function archive(EntityInterface $entity): bool
     {
-        return $this->updateStatus($entity->getId(), TaskLogInterface::STATUS_ARCHIVED);
+        return (bool)$this->updateStatus($entity->getId(), TaskLogInterface::STATUS_ARCHIVED);
```
Comment (gabrielfs7) [inline models/classes/taskQueue/TaskLog/Broker/RdsTaskLogBroker.php:215]: Did you check that any status different of 0 is positive?

File: models/classes/taskQueue/TaskLog/Broker/AbstractTaskLogBroker.php [hunk 2]
```diff
@@ -15,7 +15,7 @@
  * along with this program; if not, write to the Free Software
  * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
  *
- * Copyright (c) 2020 (original work) Open Assessment Technologies SA;
+ * Copyright (c) 2020-2021 (original work) Open Assessment Technologies SA;
```
Comment (gabrielfs7) [inline models/classes/taskQueue/TaskLog/Broker/AbstractTaskLogBroker.php:18]: Why this change?

File: models/classes/taskQueue/TaskLog/Broker/RdsTaskLogBroker.php (hunk 1 above)
Comment (boajer) [reply models/classes/taskQueue/TaskLog/Broker/RdsTaskLogBroker.php:215]: `::updateStatus` returns number of affected records, so I believe such logic is appropriate here.

Comment (gabrielfs7) [reply models/classes/taskQueue/TaskLog/Broker/RdsTaskLogBroker.php:215]: Got it, thanks!

**Review by gabrielfs7 — APPROVED** [review]
- [ ] New code is covered by tests (if applicable)
- [ ] Tests are running successfully (old and new ones) on my local machine (if applicable)
- [ ] New code is respecting code style rules
- [ ] New code is respecting best practices
- [ ] New code is not subject to concurrency issues (if applicable)
- [ ] Feature is working correctly on my local machine (if applicable)
- [ ] Acceptance criteria are respected
- [ ] Pull request title and description are meaningful

**Review by shpran — APPROVED** [review]
- [ ] New code is covered by tests (if applicable)
- [ ] Tests are running successfully (old and new ones) on my local machine (if applicable)
- [ ] New code is respecting code style rules
- [ ] New code is respecting best practices
- [ ] New code is not subject to concurrency issues (if applicable)
- [ ] Feature is working correctly on my local machine (if applicable)
- [ ] Acceptance criteria are respected
- [ ] Pull request title and description are meaningful

**Comment by AntonNeviarouski** [comment]
QA verified
